### DISCUSSION TITLE
下载怎么这么慢 ==

### DISCUSSION THREAD

**Discussion by s2ad0w** [lsky-org/lsky-pro#693] [灌水]
下载怎么这么慢 ==  
![image](https://github.com/lsky-org/lsky-pro/assets/126168851/ccb971bf-8936-4d0a-b809-f7dd8cd148d2)

**Comment by Edakerx** [comment]
lsky oss节点在香港

**Comment by akatsukiro** [comment]
腾讯云对象存储，香港。没有大陆优化，慢是正常的。似乎已经是有优化方案了，不知道多久能部署到位。
